tests/block_tools.py

```diff
@@ -88,6 +88,7 @@
 
 test_constants = DEFAULT_CONSTANTS.replace(
     **{
+        "RUST_CONDITION_CHECKER": 0,
```

Comment:
by default, tests use the rust condition checker. This is the block height at which the rust implementation is used